project/dexter-core/src/java/com/samsung/sec/dexter/core/analyzer/AnalysisResultFileManager.java

```diff
@@ -155,13 +146,17 @@ private void addOptionalContent(final StringBuilder contents, final String key,
 	private File getResultFilePath(final AnalysisResult result,
 			final String resultFolderStr) {
 		
-		final String path =  resultFolderStr + "/" + ResultFileConstant.RESULF_FILE_PREFIX + result.getFileName()
+		final String path =  resultFolderStr + "/" + getResultFilePrefixName(result.getModulePath(), result.getFileName())
 		        + "_" + DexterUtil.currentDateTimeMillis() + ResultFileConstant.RESULT_FILE_EXTENSION;
 		
 		final File resultFile = DexterUtil.createEmptyFileIfNotExist(path);
 		
 		return resultFile;
-	} 
+	}
+	
+	protected String getResultFilePrefixName(final String modulePath, final String fileName){
+		return ResultFileConstant.RESULF_FILE_PREFIX + fileName + "_" + modulePath.hashCode(); 
```

Comment:
In spite of being the other String, can there be the String having the same hashcode?